src/sql/workbench/contrib/scripting/browser/scripting.contribution.ts

```diff
@@ -329,7 +330,7 @@ MenuRegistry.appendMenuItem(MenuId.ExplorerWidgetContext, {
 			ItemContextKey.ItemType.isEqualTo('table'),
 			MssqlNodeContext.EngineEdition.notEqualsTo(DatabaseEngineEdition.SqlOnDemand.toString()),
 			ItemContextKey.ConnectionProvider.notEqualsTo('kusto'),
```

Comment:
Would you mind going through and doing the same thing for kusto as well? (we can follow up with sql after since that's used in a lot more places and is going to be harder to track down)